tests/framework/installer/ceph_manifests.go

```diff
@@ -351,7 +359,7 @@ parameters:
 	if m.settings.ConnectionsEncrypted {
 		// encryption requires either the 5.11 kernel or the fuse mounter. Until the newer
 		// kernel is available in minikube, we need to test with fuse.
-		sc += "  mounter: fuse"
+		sc += "  mounter: kernel"
```

Comment:
if someone runs the test on the lower kernel version this fails.

Reply:
> if someone runs the test on the lower kernel version this fails.

Since, the introduction of GitHub action we suggest E2E testing on GitHub action only(not locally). I think we are good here.

Reply:
> update the comments as this is always set to `kernel`

added a comment

Reply:
even if we remove this option it should ideally work because we use kernel client by default. but no harm in setting it explicitly as well.

Reply:
ahh, I'm reading this a little late. I can open a separate PR to improve this. Thanks